utilities/test_suite/HOST/README_AUDIO.md

```diff
@@ -0,0 +1,51 @@
+# AMD ROCm Performance Primitives (RPP) Test Suite
+The AMD ROCm Performance Primitives (RPP) audio test suite can be used to validate the functionality and performance of the AMD ROCm Performance Primitives (RPP) audio augmentations.
```

Comment:
@r-abishek 
Currently i have asked @HazarathKumarM to add a seperate readme for audio test suite
Please let us know if this is fine / we need to merge the audio test suite info also in image test suite

Reply:
@sampath1117 @HazarathKumarM 
No, a folder should have only one ReadMe.md
Please add two subcategories under the main header for the ReadMe page which already is - AMD ROCm Performance Primitives (RPP) Test Suite
- Calling the first one RPP Tensor Image Tests with current documentation, reducing size of subheadings by one size 
- Call the second one RPP Tensor Audio Tests with similar crystal-clear documentation on params and what the test suite supports.
Please use https://stackedit.io/ or other tool to visually see whats going on with the titles and structure if needed.